### PR TITLE
Additional guidelines on tags (#102)

### PR DESCRIPTION
**Pull request by pvdbosch**
See #102

### CODE REVIEW

File: guide/src/main/asciidoc/api_specifications.adoc [hunk 1]
```diff
@@ -49,8 +49,51 @@ According to usage, a conversion step to JSON might be necessary considering lim
...
+
+Declaring each tag allows linting tooling to detect inconsistencies in tag names.
+
+.Using tags
+====
+```YAML
+tags:
+  - name: Employers
+    description: Managing employers
+  - name: Reference data
```
Comment (JDMKSZ) [inline guide/src/main/asciidoc/api_specifications.adoc:72]: Given https://github.com/belgif/rest-guide/issues/102#issuecomment-1908294570, I would add both styles as example, with Capitalized Words as the first. So e.g. add "Employment Details" above "Reference data".

**Review by JDMKSZ — CHANGES_REQUESTED** [review]

File: guide/src/main/asciidoc/api_specifications.adoc (hunk 1 above)
Comment (jpraet) [reply guide/src/main/asciidoc/api_specifications.adoc:72]: You probably wouldn't typically mix both naming styles within the same API though?

Comment (JDMKSZ) [reply guide/src/main/asciidoc/api_specifications.adoc:72]: Good point. In that case maybe just change the example from "Reference data" to "Reference Data"?

Comment (pvdbosch) [reply guide/src/main/asciidoc/api_specifications.adoc:72]: why change? The other tag is "Employers" and the guideline doesn't prefer "Reference Data" above "Reference data".

**Comment by pvdbosch** [comment]
approved by REST WG

**Pull request merged**